kotlin/src/test/kotlin/org/jdbi/v3/core/kotlin/KotlinMapperTest.kt

```diff
@@ -40,6 +40,22 @@ class KotlinMapperTest {
 
     }
 
+    @Test fun shouldUseSecondOccurrenceOfColumnWhenMultipleColumnsWithSameNameArePresent() {
+
+        val mapper = KotlinMapper(DataClassWithOnlyPrimaryConstructor::class.java)
+        mockColumns("id", "name", "name")
```

Comment:
Apologies for the slow response--it's been an unexpectedly busy couple of weeks.